examples/fabric/v4/cloudRouterConnectivity/cloudRouter2serviceprofile/terraform.tfvars.example

```diff
@@ -1,4 +1,4 @@
-equinix_client_id      = "Equnix_Client_Id"
+equinix_client_id      = "Equinix_Client_Id"
```

Comment:
Code owner review has been enabled on this repo so we'll either need to revert this or get a 👍 from the Fabric team

Reply:
reverted in https://github.com/equinix/terraform-provider-equinix/pull/406/commits/42f2da654a27f8c93a6220fc4a2fa8bb8b0107f1 since is a typo in an example value it could be addressed later in a different PR